vassal-app/src/main/java/VASSAL/build/module/chessclockcontrol/ChessClock.java

```diff
@@ -1046,16 +1086,14 @@ public void popupMenuWillBecomeVisible(PopupMenuEvent evt) {
       });
 
       JMenuItem item;
-      final String s;
-
-      if (clockTicking) {
-        s = CHESSMENU_STOP;
+      if (isTicking()) {
+        item = new JMenuItem(CHESSMENU_STOP);
```

Comment:
If we've added a start clock hotkey, should we also add a "stop clock" hotkey while we're at it? (just a thought)

Reply:
(or maybe that's what pause-all-clocks-hotkey does, in which case maybe we should display that hotkey here?)

Reply:
May as well.  At the moment, it would be effectively identical functionally to making a call to pause-all-clocks, but there's no reason not to add support to multiple ticking clocks while we're here